apps/server/src/services/rundown-service/delayedRundown.utils.ts

```diff
@@ -115,7 +115,17 @@ export async function cachedEdit(
   }
 
   const updatedRundown = DataProvider.getRundown();
-  const newEvent = { ...updatedRundown[indexInMemory], ...patchObject } as OntimeRundownEntry;
+  const eventFromRundown = updatedRundown[indexInMemory];
+
+  const isPatchObjectSameAsRundownEvent = Object.entries(patchObject).some(
+    ([key, value]) => eventFromRundown[key] === value,
+  );
+
+  if (isPatchObjectSameAsRundownEvent) {
+    return eventFromRundown;
+  }
```

Comment:
The some will always match on the ID and therefor skip all changes
```suggestion
  const isPatchObjectDifferentFromRundownEvent= Object.entries(patchObject).some(
    ([key, value]) => eventFromRundown[key] !== value,
  );

  if (!isPatchObjectDifferentFromRundownEvent) {
    return eventFromRundown;
  }
```